modules/carbonRtdProvider.js

```diff
@@ -165,20 +178,15 @@ export function updateRealTimeDataAsync(callback, userConsent) {
     }
   }
 
-  if (features?.enableContext !== undefined) {
-    reqUrl.searchParams.append('context', features.enableContext);
-  }
-
-  if (features?.enableAudience !== undefined) {
-    reqUrl.searchParams.append('audience', features.enableAudience);
-  }
-
-  if (features?.enableCustomTaxonomy !== undefined) {
-    reqUrl.searchParams.append('custom_taxonomy', features.enableCustomTaxonomy);
-  }
+  reqUrl.searchParams.append('context', (typeof features.enableContext === 'undefined') ? true : features.enableContext);
+  reqUrl.searchParams.append('audience', (typeof features.enableAudience === 'undefined') ? true : features.enableAudience);
+  reqUrl.searchParams.append('deal_ids', (typeof features.enableDealId === 'undefined') ? true : features.enableDealId);
 
-  if (features?.enableDealId !== undefined) {
-    reqUrl.searchParams.append('deal_ids', features.enableDealId);
+  // we only want to update custom taxonomy after expiration
```

Comment:
Where is the check for `after expiration`?

Sorry if I am missing it

Reply:
It's a misleading comment, I'll update our comments to be more explicit. We only want to request updated custom taxonomy from the server if we don't have valid cached rules. That is checked on line 238 - 241.